client/my-sites/domains/domain-management/name-servers/style.scss

```diff
@@ -6,14 +6,17 @@
 
 .name-servers__dns {
 	.name-servers__toggle {
-		display: inline-block;
-		float: right;
-
-		.components-toggle-control .components-base-control__field .components-form-toggle {
-			margin-right: 0;
```

Comment:
We still need the `margin-right: 0` here - otherwise, the toggle gets pushed to the left a bit.